hs/Scripts.hs

```diff
@@ -71,7 +71,9 @@ graphScripts = do
          "/static/js/graph/sidebar/focus_descriptions.js",
          "/static/js/common/export/export.js"])
     H.script ! A.type_ "text/jsx" ! A.src "/static/js/common/react_modal.js.jsx" $ ""
-
+    H.script ! A.type_ "text/jsx" ! A.src "/static/js/graph/react_node.js.jsx" $ ""
```

Comment:
@ryanfan You can leave this for now.
@Ian-Stewart-Binks You're right, creating a new `jsxScript` function makes sense, I think.